### DISCUSSION TITLE
Can't seem to figure out how to add an ID as admin

### DISCUSSION THREAD

**Discussion by Skrakl** [EvanMulawski/BeamMP.EnvSync#17] [Q&A] [answered]
Title, more or less.
![envsettime](https://user-images.githubusercontent.com/118335561/202100198-1cd35185-419b-4382-8217-2b6ff1cee77b.png)

Naturally I've tried both ServerID and beamMP ID, still no luck. Any assistance at all would be greatly appreciated

**Comment by EvanMulawski** [comment, marked as answer]
Hi @Skrakl, did you see this section in the wiki?

https://github.com/EvanMulawski/BeamMP-EnvSync/wiki/Installation#admins

Let me know if that works for you.

**Reply by Skrakl** [reply]
Ah, I grabbed the first ID rather than the one actually called "user_ID"

Many thanks!
